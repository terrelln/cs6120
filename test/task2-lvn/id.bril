# ARGS: 42
@main(a: int): int {
    b: int = id a;
    c: int = id b;
    d: int = id c;
    e: int = id d;
    e: int = id e;
    e: int = id e;
    print e;
    ret e;
}