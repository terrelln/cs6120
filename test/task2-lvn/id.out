@main(a: int): int {
.__block0:
  print a;
  ret a;
}
